Formula/libtbox.rb

```diff
@@ -0,0 +1,38 @@
+class Libtbox < Formula
+  desc "A glib-like multi-platform c library"
+  homepage "https://tboox.org"
+  url "https://github.com/waruqi/tbox/archive/v1.5.2.tar.gz"
+  sha256 "c470a8a5b8f84d928d83af87c8f69c87ec9ecb6f89017bef93dc0d188e91a8c6"
+  head "https://github.com/waruqi/tbox.git"
+
+  depends_on "xmake" => :build
+
+  def install
+    args = ["--smallest=y"]
+    args << "--demo=n"
+    args << "--xml=y"
+    args << "--asio=y"
+    args << "--thread=y"
+    args << "--network=y"
+    args << "--charset=y"
+
+    system "xmake", "config", *args
```

Comment:
Don't need an `args` array unless it needs manipulating. Simply:
```ruby
system "xmake", "config", "--smallest=y",
                          "--demo=n",
                          "--xml=y",
                          "--asio=y",
                          "--thread=y"
```
etc, or some reasonable combination similar to that is fine.